### DISCUSSION TITLE
Lost Wallet Compensation

### DISCUSSION THREAD

**Discussion by hes91** [IBAX-io/go-ibax#412] [Ideas] [answered]
In my opinion, in order not to victimize people, to prevent loss of wallet, a recovery code should be obtained once via e-mail or sms. Thus, every person should be able to compensate for his mistake at least once. Because he may not have saved the recovery codes instantly. That's why we should be able to recover one-time wallets with digital codes given via e-mail or sms.

**Comment by ibaxio** [comment, marked as answer]
Sorry to hear about that, unfortunately we follow the principles of the blockchain so please be careful about saving your SEED or Private Key.

IBAX is a decentralized network and the system does not record your personal information (Seed, Private Key, etc) and follows this principle of privacy, therefore we cannot recover your account.

Thanks.

**Reply by hes91** [reply]
Thanks for your information
